habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -61,9 +65,16 @@ def if_config_to_lower(config):
         else:
             return config
 
+    has_ignores = ignore_keys is None
     for attr, value in config_from.items():
-        if hasattr(config_to, attr.lower()):
-            setattr(config_to, attr.lower(), if_config_to_lower(value))
+        low_attr = attr.lower()
+        if has_ignores or low_attr not in ignore_keys:
+            if hasattr(config_to, low_attr):
+                setattr(config_to, low_attr, if_config_to_lower(value))
+            else:
+                logger.warn(
+                    f"{low_attr} is not found on target {config_to} but is found on config_from. Did you make a typo in the YAML config?: {config_from}"
```

Comment:
Should this not be 
`found on {config_from}`
or else 
`found in source config`
?